patches/media-media_options.gni.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/media/media_options.gni b/media/media_options.gni
+index 0c62e9b40706d4b03cec9f00d976d4061b336b50..01561cf6ba297102fc334386f040f578b6fbbfff 100644
+--- a/media/media_options.gni
++++ b/media/media_options.gni
+@@ -9,6 +9,7 @@ import("//build/config/jumbo.gni")
+ import("//media/gpu/args.gni")
+ import("//testing/libfuzzer/fuzzer_test.gni")
+ import("//third_party/libaom/options.gni")
++if (brave_chromium_build) { import("//brave/build/config.gni") }
```

Comment:
this patch is unnecessary even without the change below because we already have a patch in chrome_build.gni